core/src/main/java/org/testcontainers/containers/SelinuxContext.java

```diff
@@ -0,0 +1,16 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.model.SELContext;
+
+/**
+ * Possible contexts for use with SELinux
+ */
+public enum SelinuxContext {
+    SHARED(SELContext.shared), SINGLE(SELContext.single), NONE(SELContext.none);
+
+    public final SELContext selContext;
```

Comment:
oh. my. word.  Where has Lombok been all my life?  This is amazing.

Although `@Value` can't be used on an enum.  Went for `@AllArgsConstructor` instead.